content/docs/examples.md

```diff
@@ -476,7 +476,7 @@ $ task all
 Do I have the latest released version?
 
 ```
-$ curl https://gothenburgbitfactory.org/latest/task
```

Comment:
The URL `https://gothenburgbitfactory.org/latest/task` should actually be redirected to `https://gothenburgbitfactory.org/task/latest` , but this no longer seems to work...

Reply:
As a bonus, I just discovered that the data on gbf.org regarding Taskwarrior is outdated... 😬

Reply:
The link redirection has been fixed with https://github.com/GothenburgBitFactory/GothenburgBitFactory.github.io/commit/179068b986f9f8d1e5863ec322a59e581575c15c

Reply:
The project data has been updated with https://github.com/GothenburgBitFactory/GothenburgBitFactory.github.io/commit/70b85338946dcac21f1de2a44fbecb22342086d4